program-test/src/lib.rs

```diff
@@ -65,6 +65,7 @@ use {
     tokio::task::JoinHandle,
 };
 // Export types so test clients can limit their solana crate dependencies
+use solana_sdk::clock::Epoch;
```

Comment:
Please add this to the common use statement above (L33)

Reply:
Fixed. Hopefully in the way you suggested.

Reply:
Fixed. Hopefully in the way you suggested.